src/tests/Interop/Interop.csproj

```diff
@@ -21,6 +21,8 @@
     <Compile Include="LayoutClass/*.cs" />
     <Compile Include="ArrayMarshalling/**/*.cs" />
     <Compile Include="MarshalAPI/**/*.cs" />
+    <!-- Disable build of GenericFunctionPointer tests on mono -->
```

Comment:
This should be a link to an issue that tracks implementing this feature in Mono.